src/cpp/core/system/Win32RuntimeLibrary.cpp

```diff
@@ -60,8 +66,6 @@ class Library : boost::noncopyable
       RS_LOAD_SYMBOL(_errno);
    }
 
-   int* (*_errno)(void);
-
```

Comment:
What was this errno doing exactly?

Reply:
I just moved that down below (to resolve a compiler warning).

It's basically a reference to the `_errno` symbol, which might be coming from either `msvcrt.dll` or `ucrtbase.dll`.